java/api/src/main/java/io/ray/api/parallelactor/ParallelActorHandle.java

```diff
@@ -0,0 +1,25 @@
+package io.ray.api.parallelactor;
+
+import io.ray.api.ActorHandle;
+
+/**
+ * A `ParallelActorHandle` could be used for creating multiple execution instances in different
+ * threads, to execute the methods.
+ *
+ * <p>Every thread holds an instance to `A`, and the methods will be invoked on the instance which
+ * is calculated according to the parallel strategy.
+ *
+ * <p>The parallel strategy are only working on the caller side, which means it doesn't be
+ * coordinated when 2 callers to invoke 1 parallel actor.
+ */
+public interface ParallelActorHandle<A> {
+
+  /** Get an execution instance of the parallel actor by the given index. */
+  ParallelInstance<A> getInstance(int index);
+
+  /** Get the parallel number of this parallel actor. */
+  int getParallelNum();
```

Comment:
Maybe `getParallelism` is better because we use `setParallelism` when creating it.